README.md

```diff
@@ -125,6 +125,8 @@ Here are the options you can set in your `.cz-config.js`:
 * **skipQuestions**: {Array of Strings: default none}. List of questions you want to skip. Eg.: ['body', 'footer'].
 * **appendBranchNameToCommitMessage**: If you use `cz-customizable` with `cz-customizable-ghooks`, you can get the branch name automatically appended to the commit message. This is done by a commit hook on `cz-customizable-ghooks`. This option has been added on `cz-customizable-ghooks`, v1.3.0. Default value is `true`.
 * **ticketNumberPrefix**: {string, default 'ISSUES CLOSED:'}: Set custom prefix for footer ticker number.
+* **ticketNumberSuffix**: {string, default ' '}: Set custom prefix for footer ticker number.
+* **ticketNumberPosition**: {string <small>[ 'first', 'standard' or 'last' ]</small>, default none}: Set custom placement for footer ticker number.
```

Comment:
what is the difference between `none` and `standard`. Maybe default could be `standard`?

Reply:
"standard" is default value here, none here means undefined. I saw this wording on lines 124, 125 of this file. And got me confused at first too...